Readme.md

```diff
@@ -48,6 +49,17 @@ app.use(function *(){
 })
 ```
 
+### setHeaders
+
+The function is called as `fn(res, path, stats)`, where the arguments are:
+* `res`: the response object
+* `path`: the resolved file path that is being sent
+* `stats`: the stats object of the file that is being sent.
+
+You should only use the `setHeaders` option when you wish to edit the `Cache-Control`, `Last-Modified` or `Content-Length` headers, because doing it before is useless (it's overwritten), and doing it after is too late because the headers are already sent.
```

Comment:
I removed Content-Length from the code and readme.

And unless I'm misunderstand, I can't validate this in my code because setHeaders is an escape hatch for the user. To validate its usage I should diff headers before/after. This would be very costly for everyone no ?